lib/common/tls.c

```diff
@@ -190,14 +188,6 @@ pcmk__init_tls(pcmk__tls_t **tls, bool server, gnutls_credentials_type_t cred_ty
 
     signal(SIGPIPE, SIG_IGN);
 
-    /* gnutls_global_init is safe to call multiple times, but we have to call
-     * gnutls_global_deinit the same number of times for that function to do
-     * anything.
-     *
-     * FIXME: When we can use gnutls >= 3.3.0, we don't have to call
-     * gnutls_global_init anymore.
-     */
-    gnutls_global_init();
     gnutls_global_set_log_level(8);
```

Comment:
While you're here, can you also add a comment to remind us to not bump the log level beyond 8, since higher levels can log sensitive information?  I should have done this when I added the code.

Reply:
Sure